techniques/silverlight/SL8.html

```diff
@@ -84,9 +84,9 @@ <h3>Displaying a long text alternative for an Image with
 &lt;/StackPanel&gt;
 </pre>
             <p>The following event handler removes the Help button and replaces
-    						it in UI with a <code class="obj">TextBox</code> that displays the long text
+    						it in UI with a <code>TextBox</code> that displays the long text
     						alternative. </p>
-            <pre xml:space="preserve">       private void HelpButton_Click(object sender, RoutedEventArgs e)
+            <pre>       private void HelpButton_Click(object sender, RoutedEventArgs e)
```

Comment:
```suggestion
            <pre><code>       private void HelpButton_Click(object sender, RoutedEventArgs e)
```

and add matching `</code>`